Doc/library/pathlib.rst

```diff
@@ -1055,3 +1071,42 @@ call fails (for example because the path doesn't exist):
       'Text file contents'
 
    .. versionadded:: 3.5
+
+Correspondence to tools in the :mod:`os` package
+------------------------------------------------
+
+If you're more familiar with :mod:`os.path` module, here's a correspondence
```

Comment:
```
Below is a table mapping various :mod:`os` functions to their corresponding :class:`PurePath`/:class:`Path` equivalent.
```
Basically the documentation doesn't take quite such a casual, conversational tone.